.eslintrc.json

```diff
@@ -1,25 +1,15 @@
 {
-  "env": {
-    "browser": true,
-    "es6": true,
-    "jest": true
-  },
-  "parserOptions": {
-    "ecmaVersion": 2018,
-    "sourceType": "module"
-  },
-  "extends": ["airbnb-base"],
-  "rules": {
-    "no-shadow": "off",
-    "no-param-reassign": "off",
-    "eol-last": "off",
-    "arrow-parens": "off",
-    "no-alert":"off",
-    "quotes": ["error", "double"],
-    "no-use-before-define": ["error", {
-        "functions": false,
-        "classes": false,
-        "variables": false
- }]
-  }
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es2020": true
```

Comment:
- [ ] Eslint it's not recognizing Jest variables, we can fix that adding it to the env object here. `"jest": true`